lib/elements/rebar.rb

```diff
@@ -0,0 +1,66 @@
+require './db/base'
+require './lib/materials/steel'
+
+module Elements
+  class Rebar < Materials::Steel
+    MAX_YIELD_STRESS = 550.to_f
+
+    attr_reader :diameter, :yield_stress, :number, :hooks_class, :elastic_module, :start_hook, :end_hook
+
+    def self.build_by_standard(number:, yield_stress:)
+      rebar_data = DB::Base::STANDARD_REBAR.find { |bar_data| bar_data['number'] == number }
+
+      @number = number
```

Comment:
I think this should be set in the constructor

Reply:
yeah, done!